app/styles/components/c-the-project.scss

```diff
@@ -0,0 +1,132 @@
+@import '../application';
+
+.c-the-project {
```

Comment:
Can you add a comment saying in a nearby future we should unify all static texts in one component? Thanks!